sdk/invoke.js

```diff
@@ -21,10 +21,25 @@ const invoke = async function invoke(funcName, apiKey, funcData) {
     resolveWithFullResponse: true,
   };
   logger.debug('Invoking function', options);
-  const res = await rp.post(options);
-  const { statusCode, headers, body } = res;
-  logger.debug('Invoke response', { statusCode, headers, body });
-  return res;
+  try {
+    const res = await rp.post(options);
+    const { statusCode, headers, body } = res;
+    logger.debug('Invoke response', { statusCode, headers, body });
+    return res;
+  } catch (err) {
+    let parsedError;
+    try {
+      parsedError = JSON.parse(err.error);
+    } catch (nestedErr) {
+      throw new Error(err.error || err);
+    }
+    if (parsedError.stack && (parsedError.error || parsedError.message)) {
+      throw new Error(`${parsedError.error || parsedError.message}\n${parsedError.stack}`);
+    } else if (parsedError.message && parsedError.request_id) {
```

Comment:
why `else` here and not in line 41. please be consistent.
(our common style is to avoid the else)

Reply:
Can you explain the rationale behind not using else? I can maybe understand if this were C and we had to think of branch prediction or something.